addons/package_hierarchy/views/stock_picking_views.xml

```diff
@@ -5,15 +5,10 @@
         <field name="inherit_id" ref="stock.view_picking_form"/>
         <field name="model">stock.picking</field>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='move_line_ids']//field[@name='product_id']" position="before">
-                <field name="x_selected" />
-            </xpath>
-
-            <xpath expr="//button[@name='put_in_pack']" position="before">
-                <field name="x_pallet_id" />
-                <button name="palletise" type="object" string="Palletise"/>
-            </xpath>
-
+            <!-- Two enters meant for removing the put_in_pack buttons
```

Comment:
Think this should be "two entries". Is there not a way to replace both instances in one line? (I assume that the way this works is to find the first instance of the button and replace it and then find the next instance and replace that?) I'm not overly familiar with xpath to know.

Reply:
Ofc :)
Nope, I'm afraid not as it's due to the template magic that seems to iterate a collection of items without inner references.

Reply:
cool! =)